src/dotnet/APIView/APIView/TreeRendererAPIV.cs

```diff
@@ -416,6 +428,31 @@ public void Render(PropertyAPIV p, StringBuilder builder, int indents = 0)
             builder.Append("}");
         }
 
+        public void Render(TokenAPIV token, StringBuilder builder)
```

Comment:
I wonder if this entire method should be abstract. In text renderer it would just output display string and in html renderer - switch the span class.